masonry_core/src/core/contexts.rs

```diff
@@ -151,7 +160,9 @@ impl_context_method!(
     ComposeCtx<'_>,
     PaintCtx<'_>,
     AccessCtx<'_>,
+    RawCtx<'_>,
     {
+        #[allow(dead_code, reason = "Copy-pasted for some types that don't need it")]
```

Comment:
I don't see how this can ever be dead_code; the types are all pub, and it's a pub method

Reply:
Yup, that was a copy-pasting error.